### PR TITLE
Fix log numbber incorrect on pulsar metadata

### PR DESCRIPTION
**Pull request by Shoothzj**
### Modifications

Fix log numbber incorrect on pulsar metadata

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 
  
- [x] `no-need-doc` 
  
  Simple changes, no need doc

### CODE REVIEW

**Review by Jason918 — APPROVED** [review]
LGTM

File: pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/PulsarLedgerIdGenerator.java [hunk 1]
```diff
@@ -226,7 +226,7 @@ private String formatHalfId(int i) {
                     // delete the znode for id generation
                     store.delete(stat.getPath(), Optional.empty()).
                             exceptionally(ex -> {
-                                log.warn("Exception during deleting node for id generation : {}", ex);
+                                log.warn("Exception during deleting node for id generation: ", ex);
```
Comment (yuruguo) [inline pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/PulsarLedgerIdGenerator.java:229]: same as above

File: pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/PulsarLedgerIdGenerator.java [hunk 2]
```diff
@@ -95,7 +95,7 @@ public void generateLedgerId(BookkeeperInternalCallbacks.GenericCallback<Long> g
...
```
Comment (yuruguo) [inline pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/PulsarLedgerIdGenerator.java:98]: Is there no need to add `{}` here?

**Review by hangc0276 — APPROVED** [review]

**Review by 315157973 — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**